topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to recurrent neural networks (RNN)
+zenodo_link: https://zenodo.org/record/4477881
+questions:
+- What is a recurrent neural network (RNN)?
+- What are some applications of RNN?
+objectives:
+- Understand the difference between feedforward neural networks (FNN) and RNN
+- Learn various RNN types and architectures
+- Learn how to create a neural network using Galaxy's deep learning tools
+- Solve a sentiment analysis problem on IMDB movie review dataset using RNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+
+---
+
+
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Artificial neural networks are a machine learning discipline roughly inspired by how neurons in a 
+human brain work. In the past decade, there has been a huge resurgence of neural networks thanks 
+to the vast availability of data and enormous increases in computing capacity (Successfully 
+training complex neural networks in some domains requires lots of data and compute capacity). There 
+are various types of neural networks (Feedforward, recurrent, etc). In this tutorial, we discuss 
+recurrent neural networks, which model sequential data, and have been successfully applied to language 
+generation, machine translation, speech recognition, image description, and text summarization 
+({% cite wen-etal-2015-semantically %}, {% cite cho-etal-2014-learning %}, {% cite LimEtAl %}, 
+{% cite karpathyEtAl %}, {% cite li-etal-2017-deep %}). We then explain how RNN differ from feedforward networks, 
+describe various RNN architectures and solve a sentiment analysis problem using RNN in Galaxy.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Review of feedforward neural networks (FNN)
+
+In feedforward neural networks (FNN) a single training example is presented to the network, 
+after which the the network generates an output. For example, a lung X-ray image is passed 
+to a FNN, and the network predicts tumor or no tumor. By contrast, in RNN a sequence of 
+training examples are presented to the network one at a time. For example, a sequence of 
+English words is passed to a RNN, one at a time, and the network generates a sequence of 
+Persian words, one at a time. RNN handle sequential data, whether its temporal or ordinal.
+
+## Single layer FNN
+
+![Alternative text](../../images/FFNN_no_hidden.png "Single layer feedforward neural network")
```

Comment:
Could you please supply some more detailed alt-text here? Here's an example of what is appropriate in each field.

Field | Example
--- | --- 
alt text |  Image of cell membrance with an embedded protein with central pore. DNA is shown splitting and entering the pore, an electrical signal comes out reading A C T or G.
fig caption | Using nanopore sequencing, a single molecule of DNA or RNA can be sequenced without the need for PCR amplification or chemical labeling of the sample. (Image from: <a href="https://nanoporetech.com/sites/default/files/s3/white-papers/WGS_Assembly_white_paper.pdf?submissionGuid=40a7546b-9e51-42e7-bde9-b5ddef3c3512">Nanopore sequencing: The advantages of long reads for genome assembly</a>)

And some more comments here https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text

Reply:
This is very important for accessibility, so vision impaired and blind users can access the content as well.

Reply:
Added alt text to all figures in the tutorial.